### ISSUE TITLE
Attribute acting weird/not working properly

### ISSUE THREAD

**Issue by denlille** [team-abnormals/savage-and-ravage#154]
Hi, I'm not sure that the attribute works correctly.
For example if I check the explosive damage reduction attribute while wearing the full griefer armor, it still says that my entity has no explosive damage reduction.

When using this attribute with an armor set's mod, it input these errors "Failed to find attribute savage_and_ravage:explosive_damage_reduction"
Other modded attribute works.

(it's the same for the stealth attribute coming from environmental btw)

**Comment by bageldotjpg** [comment]
Correct, these are not part of any entity's base attributes. They are part of the armor's attributes
